manifests/bosh-manifest/deployments/aws/090-bosh-aws-stub.yml

```diff
@@ -21,7 +21,6 @@ meta:
 
   bosh_private_ip: (( grab terraform_outputs.microbosh_static_private_ip ))
```

Comment:
Yes we can, but I'm not sure if it's a part of this story to clean up bosh manifest. I noted this in our pipeline improvements document.